src/main/java/com/vadmack/mongodbtest/service/ProjectService.java

```diff
@@ -55,4 +58,11 @@ public void delete(Long id) {
         Optional<Project> optionalProject = repository.findById(id);
         optionalProject.ifPresent(project -> repository.delete(project));
     }
+
+    private void validateDto(ProjectNoIdDto dto) {
```

Comment:
Please try to implement the ticket using following approach:
https://www.javadevjournal.com/spring/spring-rest-api-validation/